essentials-framework/Essentials DM/Essentials_DM/Chassis/DmBladeChassisController.cs

```diff
@@ -563,14 +564,51 @@ void Chassis_DMOutputChange(Switch device, DMOutputEventArgs args)
             var outCard = input == 0 ? null : Chassis.Outputs[output];
 
             // NOTE THAT BITWISE COMPARISONS - TO CATCH ALL ROUTING TYPES 
-            if ((sigType | eRoutingSignalType.Video) == eRoutingSignalType.Video) {
-                Chassis.VideoEnter.BoolValue = true;
-                Chassis.Outputs[output].VideoOut = inCard;
+            if ((sigType | eRoutingSignalType.Video) != eRoutingSignalType.Video) return;
+            Chassis.VideoEnter.BoolValue = true;
+            Chassis.Outputs[output].VideoOut = inCard;
+        }
+
+        #endregion
+
+        #region IRoutingNumeric Members
+
+        public void ExecuteNumericSwitch(ushort inputSelector, ushort outputSelector, eRoutingSignalType sigType)

```

Comment:
Make this method just call the existing ExecuteSwitch method.

Reply:
Yeah, not sure why I didn't think about this.  That's what I get for trawling issues at nearly midnight 😄